en/guide/mavlink_version.md

```diff
@@ -65,6 +65,21 @@ However, an additional heartbeat could be an issue for deployed systems.
 Therefore these radios can alternatively confirm their *MAVLink 2* compliance by emitting `RADIO_STATUS` in v2 message format after receiving the first MAVLink v2 frame.
 
 
+## Versions and Signing
```

Comment:
@benbarkay @julianoes This is supposed to capture the essence of #144. Basicaly making it clear what a channel is, and how versions/signing are contingent on that. This leads into the negotiation section, which I hope makes more sense once you know these limitations.